spec/js/views/todo_view_spec.js

```diff
@@ -40,4 +39,33 @@ define([
       expect(instance.$el.find('.delete').length).toBe(expectedLength);
     });
   });
+
+  describe('Check item', function () {
+    beforeEach(function () {
+      subject = function () {
+        instance.render();
+        instance.toggleCompleteStatus();
+      };
+    });
+
+    it('executes call of function', function () {
+      var spy = spyOn(instance, 'toggleCompleteStatus').and.callThrough();
+
+      subject();
+
+      expect(spy).toHaveBeenCalled();
+    });
+
+    it('model complete status is modified to true', function () {
+      subject();
```

Comment:
This test and the one that follow re-test the `Todo#toggleCompleteStatus` method.